web/src/pages/news/blogs/[slug].page.tsx

```diff
@@ -21,28 +21,47 @@ import {
 
 import type { GetServerSidePropsContext } from "next";
 
-interface BlogPageProps {
-	story: ISbStoryData<BlogPostStoryblok>;
+export type BlogPageErrorProps = {
+	error: string;
+};
+
+export type BlogPageSuccessProps = {
+	story: StoryblokStory<BlogPostStoryblok>;
 	breadcrumbs?: Breadcrumb[];
-}
+};
 
-export default function BlogPostPage({
-	story,
-	breadcrumbs,
-}: BlogPageProps): React.ReactElement {
-	const additionalMetaTags = useMemo(
-		() => getAdditionalMetaTags(story),
-		[story]
-	);
+export type BlogPageProps = BlogPageSuccessProps | BlogPageErrorProps;
 
-	setComponents({
-		blogPost: StoryblokBlogPost,
-		quote: Blockquote,
-		youtubeEmbed: StoryblokYoutubeEmbed,
-		iframe: StoryblokIframe,
-	});
+//TODO check if moving this out of the component improves performance. Or, should this be in the component function and wrapped in a useMemo/useCallback?
```

Comment:
Same here - would prefer it if this is captured in the tech debt spreadsheet rather than a comment so that it doesn't get lost